specifications/catalog/example/dcat.distribution.example.json

```diff
@@ -0,0 +1,84 @@
+{
+    "@context":  "https://w3id.org/idsa/v5/context.json",
+    "@id": "http://provider.com/catalog1/dataset1/",
+    "@type": "dcat:Dataset",
+    "dct:title" : "IDS Dataset #1",
+    "dct:description": [ {
+        "@language": "en",
+        "@value": "This is the number 1 data asset of the IDS. The consumer must be a TRUST_PLUS certfified connector."
+    }, {
+        "@language": "de",
+        "@value": "Dies ist das Nummer 1 Data Asset im IDS. Der Consumer muss ein TRUST_PLUS zertifizierter Connetor sein."
+    } ],
+    "dcat:keyword": [
+        "IDS",
+        "data asset",
+        "dcat dataset"
```

Comment:
Maybe we should use other key word tags since those can be inferred from the context?

Reply:
Have made up two other terms: "high value data", "example"